src/platform/site-wide/sass/shame.scss

```diff
@@ -351,6 +367,12 @@ $medium-phone-screen: 375px;
       padding-left: 1em;
       padding-right: 1em;
     }
+
+    .va-nav-breadcrumbs.new-grid,
+    article.new-grid {
```

Comment:
Would we be setting all articles have to 0 left and right padding once the new grid is in place across all pages? I'm not sure if we use article in places other than where you added `new-grid`, but that might be worth checking.

Reply:
Instead of setting the padding to `0`, we'd just remove the selectors altogether… both the `new-grid` and the original. That should be the same thing as setting to `0`, but I don't think it will require the rule.